pkg/cache/export_cache_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func TestUnmarshalExportCache(t *testing.T) {
```

Comment:
I think we need another case where ExportCache is neither a single item or a list so we have a error trying to unmarshal it

Reply:
`Test_ExportCacheUnmarshalYAML_WithError` is the one you're looking for :)